pallets/did/src/tests.rs

```diff
@@ -165,112 +161,6 @@ fn check_successful_simple_ecdsa_creation() {
 		});
 }
 
-#[test]
-fn check_successful_complete_creation() {
```

Comment:
How come we don't need a replacement for this test?

Reply:
We don't have such a thing anymore. The complete creation was a creation with all keys and service endpoints, but not the creation only includes the authentication key.